user/views.py

```diff
@@ -1,3 +1,68 @@
+import datetime
+import json
+import bcrypt
+import jwt
+
 from django.shortcuts import render
+from django.views     import View  
+from django.http      import HttpResponse, JsonResponse
```

Comment:
```suggestion
from django.http      import (
    HttpResponse, 
    JsonResponse
)
```

한 줄에서 import 하고자 하는 대상이 두 개 이상일 경우 개별 줄로 분리해주는 것이 위코드 코딩 컨벤션입니다 :-)